File: Core/Models/RealEstate.cs
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace real_estate_market.Core.Models
{
    [Table("RealEstates")]
    public class RealEstate
    {
        public int Id { get; set; }
        public float Area { get; set; }
        [Required]
        public int Level { get; set; }
        public int NumberOfRooms { get; set; }
        [Required]
        public string Address { get; set; }
        public Cladding Cladding { get; set; }
        public int CladdingId { get; set; }
        public float Price { get; set; }
        public ICollection<Photo> Photos { get; set; }
        public RealEstate()
        {
            Photos = new Collection<Photo>();
        }
    }
}

File: ClientApp/src/app/models/realestate.ts
export interface KeyValuePair { 
  id: number; 
  name: string; 
}

export interface RealEstate {
  id: number; 
  
}

export interface SaveRealEstate {
  id: number; 
  claddingId: number;
  area: number;
  level: number;
  numberOfRooms: number;
  price: number;
  address: string;
  isBooked: boolean;
}